app/assets/javascripts/cardboard/new_tab_links.js

```diff
@@ -1,8 +1,8 @@
-$(document).on("page:update", function() {
```

Comment:
It appears that the merge from master didn't remove this file, so I'm going to take it out manually. We decided to axe the functionality [here](https://github.com/smashingboxes/cardboard/pull/123).